iceoryx_dds/examples/docker/ice_publisher_gateway.entrypoint

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+#
+#/ Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+/usr/bin/iox-roudi &
+/usr/bin/iox-gw-iceoryx2dds &
+/usr/bin/ice-publisher-simple 
```

Comment:
okay, I didn't knew the examples can also be installed. In this case I think they should also have a prefix, maybe `iox-ex-`. What do you think?